modules/templates/util_json.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package templates
+
+import (
+	"bytes"
+
+	"code.gitea.io/gitea/modules/json"
+)
+
+type JsonUtils struct{} //nolint:revive
```

Comment:
What was revive complaining about?

Reply:
JSON ....

Reply:
Huh, should probably disable such silly naming requirements.